redux/README.md

```diff
@@ -0,0 +1,34 @@
+# Managing application state - redux
+
+## Agenda
+
+1. Problem introduction
+  - Components are [too long](https://github.com/msd-code-academy/06-cinema-portal-redux/blob/master/src/PurchasePage.js)
```

Comment:
Good thinking, but we want to change master a bit and this file will contain some new TODOs for participants.